spec/services/support_interface/notes_export_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::NotesExport do
```

Comment:
All exports should now include `it_behaves_like 'a data export'` in the specs. See the other export specs for an example of how this is done. 

https://github.com/DFE-Digital/apply-for-teacher-training/blob/2677f58aaaa9679494616156d9b60a2ec8190900/spec/services/support_interface/sites_export_spec.rb#L13